service/src/test/java/uk/gov/hmcts/reform/fpl/service/DraftCMOServiceTest.java

```diff
@@ -149,16 +171,40 @@ void shouldReturnEmptyStringIfOthersDoesNotExist() {
         assertThat(othersKey).isEqualTo("");
     }
 
-    private DynamicList getDynamicList() {
-        DynamicList dynamicList = draftCMOService.buildDynamicListFromHearingDetails(createHearingBookings(date));
+    @Test
+    void shouldReturnAMapWithAllIndividualCMOEntriesPopulated() {
+        caseManagementOrder = CaseManagementOrder.builder()
+            .hearingDate(formatLocalDateToMediumStyle(2))
+            .id(fromString("6b3ee98f-acff-4b64-bb00-cc3db02a24b2"))
+            .recitals(List.of(Element.<Recital>builder()
+                .value(Recital.builder().build())
+                .build()))
+            .schedule(Schedule.builder().build())
+            .cmoStatus(SELF_REVIEW)
+            .orderDoc(DocumentReference.builder().build())
+            .build();
```

Comment:
Could this data creation be extracted, always a bit noisy to see something like that in tests.

Reply:
extracted to method